mmedit/models/components/refiners/mlp.py

```diff
@@ -0,0 +1,58 @@
+import torch.nn as nn
+from mmcv.runner import load_checkpoint
+
+from mmedit.models.registry import COMPONENTS
+from mmedit.utils import get_root_logger
+
+
+@COMPONENTS.register_module()
+class MLP(nn.Module):
+    """Multilayer perceptrons (MLPs).
+
+    Args:
+        in_dim (int): Input dimension.
+        out_dim (int): Output dimension.
+        hidden_list (list[int]): List of hidden dimensions.
+    """
+
+    def __init__(self, in_dim, out_dim, hidden_list):
+        super().__init__()
+        layers = []
+        lastv = in_dim
+        for hidden in hidden_list:
+            layers.append(nn.Linear(lastv, hidden))
+            layers.append(nn.ReLU())
+            lastv = hidden
+        layers.append(nn.Linear(lastv, out_dim))
+        self.layers = nn.Sequential(*layers)
+
+    def forward(self, x):
+        """Forward function.
+
+        Args:
+            x (Tensor): The input of MLP.
+
+        Returns:
+            Tensor: The output of MLP.
+        """
+        shape = x.shape[:-1]
+        x = self.layers(x.view(-1, x.shape[-1]))
+        return x.view(*shape, -1)
+
+    def init_weights(self, pretrained=None, strict=True):
+        """Init weights for models.
+
+        Args:
+            pretrained (str, optional): Path for pretrained weights. If given
+                None, pretrained weights will not be loaded. Defaults to None.
+            strict (boo, optional): Whether strictly load the pretrained model.
+                Defaults to True.
+        """
+        if isinstance(pretrained, str):
+            logger = get_root_logger()
+            load_checkpoint(self, pretrained, strict=strict, logger=logger)
+        elif pretrained is None:
+            pass
```

Comment:
Why it is passed

Reply:
Because `pretrained is None` means no pretrained model for MLP and there is no special parameter initialization method for MLP.

Reply:
Okay

Reply:
@nbei what do you think